commands/core/core.drush.inc

```diff
@@ -97,6 +97,7 @@ function core_drush_command() {
     ),
     'options' => array(
       'entity-updates' => 'Run automatic entity schema updates at the end of any update hooks. Defaults to --no-entity-updates.',
+      'post-updates' => 'Run hook_post_update_X() implementations. Defaults to --post-updates.',
```

Comment:
Better to say "Use --post-updates=0 to bypass post-updates.".

Also needs PR for master branch. Thanks.